packages/zowe-explorer/src/Profiles.ts

```diff
@@ -424,7 +422,8 @@ export class Profiles extends ProfilesCache {
             return;
         }
         const editSession = this.loadNamedProfile(profileLoaded.name, profileLoaded.type).profile;
-        const editURL = editSession.host + ":" + editSession.port;
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+        const editURL = `${editSession.host}:${editSession.port}`;
```

Comment:
Curious if `editSession.port as string` would eliminate the need for the ESLint disable comment?

Reply:
I wouldn't be opposed to that, just didn't know how we felt about casting to string in case `port` happens to be a number.